tracing-subscriber/src/fmt/mod.rs

```diff
@@ -7,18 +7,101 @@
 //! implementation of the [`Subscriber`] trait that records `tracing`'s `Event`s
 //! and `Span`s by formatting them as text and logging them to stdout.
 //!
+//! ## Usage
+//!
+//! First, add this to your `Cargo.toml` file:
+//!
+//! ```toml
+//! [dependencies]
+//! tracing-subscriber = "0.1"
```

Comment:
Nit: this should probably be
```suggestion
//! tracing-subscriber = "0.2"
```
since the next `tracing-subscriber` release will be v0.2, and the features these docs describe won't be present in v0.1.x.